renovate.json5

```diff
@@ -0,0 +1,61 @@
+{
+  "extends": ["apollo-open-source", ":semanticCommits"],
```

Comment:
Semantic is actually set upstream in aos: https://github.com/apollographql/renovate-config-apollo-open-source/blob/master/package.json

(Search for `semantic`)